src/pybind/mgr/cephadm/templates/services/keepalived/keepalived.conf.j2

```diff
@@ -0,0 +1,30 @@
+# {{ cephadm_managed }}
+vrrp_script check_haproxy {
+    script "curl http://localhost:{{ spec.ha_proxy_port }}/haproxy_test"
+    weight -20
+    interval 2
+    rise 2
+    fall 2
+}
+
+vrrp_instance VI_0 {
+  state {{ state }}
+  priority 100
+  interface {{ spec.virtual_ip_interface }}
+  virtual_router_id 51
+  advert_int 1
+  authentication {
+      auth_type PASS
+      auth_pass {{ spec.keepalived_password }}
```

Comment:
We do not need to use a "keepalived user":

The authentication block specifies the authentication type (auth_type) and password (auth_pass) used to authenticate SERVERS for failover synchronization. PASS specifies password authentication; 

So we only need "auth_pass" to allow keepalived servers to communicate between them.